india_banking/india_banking/doctype/bank_connector/bank_connector.py

```
@@ -744,6 +780,32 @@ def get_bank_statements(
 				indicator="red",
 			)
 
+	def handle_same_site_request(self, payload):
+		self.validate_connector_app_installed()
+		from india_banking_connector.api import connect
+		from india_banking_connector.utils import ResponseObject
+
+		try:
+			response = connect(**payload)
+			response = ResponseObject(response, 200)
+		except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)